validation/jinja2/validation/_wrong_buttons.html

```diff
@@ -17,16 +17,17 @@
 
 {% macro button_col(recording, csrf_token, request) %}
 
-    <button class="button {{ 'button--neutral-solid' if recording.wrong_speaker else 'button--neutral' }} wrong-speaker-button" data-cy="wrong-button" data-toggle="modal" data-target="#{{ recording.id }}-modal" data-rec-id={{ recording.id }}>Wrong Speaker</button>
-    <button class="button {{ 'button--neutral-solid' if recording.wrong_word else 'button--neutral' }} wrong-word-button" data-cy="wrong-button" data-rec-id={{ recording.id }} onclick="showWrongWordDiv('{{ recording.id }}')">Wrong Word</button>
+    <button class="button {{ 'button--neutral-solid' if recording.wrong_speaker else 'button--neutral' }} wrong-speaker-button" data-cy="wrong-speaker-button" data-toggle="modal" data-target="#{{ recording.id }}-modal" data-rec-id={{ recording.id }}>Wrong Speaker</button>
+    <button class="button {{ 'button--neutral-solid' if recording.wrong_word else 'button--neutral' }} wrong-word-button" data-cy="wrong-word-button" data-rec-id={{ recording.id }}>Wrong Word</button>
+
     <div class="menu__none rec-wrong-word" data-rec-id={{ recording.id }}>
         <form id="{{ recording.id }}-wrong-word" method="POST" action="/api/save_wrong_word/{{ recording.id }}">
             <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
             <input type="hidden" name="referrer" value="{{ request.build_absolute_uri() }}">
             What word is being spoken?
             <input type="text" id="wrong_word" name="wrong_word">
             <input form="{{ recording.id }}-wrong-word" type="submit" value="Save" class="button button--success button--small save-wrong-word" data-rec-id={{ recording.id }}>
-            <input type="button" class="button button--fail button--small" value="Cancel" data-rec-id={{ recording.id }})" onclick="hideWrongWordDiv('{{ recording.id }}')">
+            <input type="button" class="button button--fail button--small" value="Cancel" data-cy="cancel-wrong-word-button" data-rec-id={{ recording.id }})">
```

Comment:
Should this be a `<button>`?

```suggestion
            <button type="button" class="button button--fail button--small" data-cy="cancel-wrong-word-button" data-rec-id={{ recording.id }})">Cancel</button>
```